test/integration/users.test.js

```diff
@@ -1229,4 +1231,80 @@ describe("Users", function () {
         });
     });
   });
+
+  describe("POST /users/migrate", function () {
+    let fetchStub;
+
+    beforeEach(async function () {
+      fetchStub = Sinon.stub(axios, "get");
+    });
+
+    afterEach(async function () {
+      Sinon.restore();
+    });
+
+    it("Should update the user", async function () {
```

Comment:
i think the title must be should add github_id to the user

Reply:
Updated the title